src/utils/websockets.js

```diff
@@ -1,11 +1,13 @@
 import SockJS from 'sockjs-client';
 import { Client } from '@stomp/stompjs';
-
+import { noConnection } from '../actions/WindowActions';
+import { store } from '../containers/App';
```

Comment:
I really don't like that. Can't we pass `store` in params to `connectWS` instead ?

Reply:
But why to add an extra param to connectWS ? Doesn't have enough params already ? If we were to do that that would mean passing it along in every call. Just do a quick search by connectWS and see where this is called. We don't want to complicate things right ? Instead as I did it I exported the store in one place and import it in here. It's simple and doesn't imply making sure I am passing correct store in each of the files where it should be called (DocumentList, DraggableWrapper, Board, MasterWindow). So logic is kept is kept simple and not spread across multiple files.